### PR TITLE
Script update and documentation update for ldap testing

### PR DESCRIPTION
**Pull request by SagarGi**

### CODE REVIEW

File: ldap-testing/README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+bash -x ldap-testing/start.sh
+````
+
+Also to stop LDAP server and LDAPAdmin run the stop.sh script
+```
+bash -x ldap-testing/stop.sh
+````
+
+
+For more details visit (https://github.com/owncloud/core/wiki/How-to-test-user_ldap).
```
Comment (kiranparajuli589) [inline ldap-testing/README.md:16]:
```suggestion
For detailed information please visit https://github.com/owncloud/core/wiki/How-to-test-user_ldap.
```

File: ldap-testing/start.sh [hunk 2]
```diff
@@ -11,16 +11,17 @@ LDAP_ROOTPASS=admin
...
-	-e SLAPD_ADDITIONAL_MODULES=memberof \
-	--name docker-slapd \
-	-d dinkel/openldap || exit 1
+docker run --rm \
+    -e LDAP_TLS_VERIFY_CLIENT=never \
+    -e LDAP_DOMAIN=owncloud.com \
+    -e LDAP_ORGANISATION=ownCloud \
+    -e LDAP_ADMIN_PASSWORD=admin \
+    --name docker-slapd \
+    -p 127.0.0.1:389:389 \
```
Comment (kiranparajuli589) [inline ldap-testing/start.sh:23]:
```suggestion
    -p 127.0.0.1:$LDAP_LOCAL_PORT:389 \
```

File: ldap-testing/README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
-Visit `https://github.com/owncloud/core/wiki/How-to-test-user_ldap` for more details
+
+
+Clone [administration](https://github.com/owncloud/administration) repo and run `start.sh` to start a LDAP server and LDAPAdmin
+
+```
```
Comment (kiranparajuli589) [inline ldap-testing/README.md:5]:
```suggestion
```bash
```

File: ldap-testing/start.sh [hunk 4]
```diff
@@ -11,16 +11,17 @@ LDAP_ROOTPASS=admin
...
-docker run -p 127.0.0.1:$LDAP_LOCAL_PORT:389 \
-	-e SLAPD_DOMAIN=$LDAP_DOMAIN \
-	-e SLAPD_ORGANIZATION="$LDAP_ORGANISATION" \
-	-e SLAPD_PASSWORD=$LDAP_ROOTPASS \
-	-e SLAPD_ADDITIONAL_MODULES=memberof \
-	--name docker-slapd \
-	-d dinkel/openldap || exit 1
+docker run --rm \
+    -e LDAP_TLS_VERIFY_CLIENT=never \
+    -e LDAP_DOMAIN=owncloud.com \
```
Comment (phil-davis) [inline ldap-testing/start.sh:19]:
```suggestion
    -e LDAP_DOMAIN=$LDAP_DOMAIN \
```

File: ldap-testing/start.sh [hunk 5]
```diff
@@ -11,16 +11,17 @@ LDAP_ROOTPASS=admin
...
+    -e LDAP_ORGANISATION=ownCloud \
```
Comment (phil-davis) [inline ldap-testing/start.sh:20]:
```suggestion
    -e LDAP_ORGANISATION=$LDAP_ORGANISATION \
```

**Review by phil-davis — COMMENTED** [review]
LDAP_DOMAIN and LDAP_ORGANISATION have definitions at lines 7 and 8 already, so we should keep using those.

File: ldap-testing/start.sh [hunk 6]
```diff
@@ -11,16 +11,17 @@ LDAP_ROOTPASS=admin
...
+    -e LDAP_ADMIN_PASSWORD=admin \
```
Comment (kiranparajuli589) [inline ldap-testing/start.sh:21]:
```suggestion
    -e LDAP_ADMIN_PASSWORD=$LDAP_ROOTPASS \
```

**Review by phil-davis — APPROVED** [review]

**Pull request merged**